schemas/com.snowplowanalytics.monitoring.batch/alert/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Webhook payload describing an alert coming from RDB Loader",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.batch",
+    "name": "alert",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "folder": {
+      "description": "A folder that caused an alert",
+      "type": "string",
+      "format": "uri",
+      "maxLength": 1024
+    },
+    "version": {
+      "type": "string",
+      "description": "Application's version"
```

Comment:
just a nit: an uninitiated person would need to check top description to understand that it is rdb loader being referred by `Application`